dockers/docker-fpm-frr/base_image_files/TSA

```diff
@@ -54,8 +60,10 @@ fi
 /usr/bin/TS TSA
 if [[ "$(sonic-cfggen -d -v DEVICE_METADATA.localhost.type)" == *"SpineRouter"* ]] ; then
```

Comment:
Check if the DEVICE_METADATA.localhost.type stays are "SpineRouter" ?

Reply:
@judyjoseph - these checks will be updated as part of a different PR.